packages/widget.vite-plugin.unstable/src/dev-plugin/FoundryWidgetDevPlugin.ts

```diff
@@ -64,6 +67,20 @@ export function FoundryWidgetDevPlugin(): Plugin {
       htmlEntrypoints = getInputHtmlEntrypoints(options);
     },
 
+    /**
+     * Check that the FOUNDRY_TOKEN environment variable is set when running in dev mode. This is
+     * required to publish the widget overrides to Foundry.
+     */
+    config(_, { command }) {
+      if (command === "serve" && process.env.VITEST == null) {
```

Comment:
What are the two checks here for? It sounds like "serve" means "dev" for backwards compatibility and is VITEST for this repo only during tests?